catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/notes/NoteUtil.java

```diff
@@ -0,0 +1,139 @@
+        } else {
+            return null;
+        }
+        if (metacard.getAttribute(Note.PARENT_ID) != null) {
+            requiredAttributes.put("parent",
+                    (String) metacard.getAttribute(Note.PARENT_ID)
+                            .getValue());
+        } else {
+            return null;
+        }
+        if (metacard.getCreatedDate() != null) {
+            requiredAttributes.put("created",
+                    metacard.getCreatedDate()
+                            .toString());
+        }
+        if (metacard.getModifiedDate() != null) {
+            requiredAttributes.put("modified",
+                    metacard.getModifiedDate()
+                            .toString());
+        }
+        if (metacard.getAttribute(Note.COMMENT) != null) {
+            requiredAttributes.put("note",
+                    (String) metacard.getAttribute(Note.COMMENT)
+                            .getValue());
+        } else {
+            return null;
+        }
+        if (metacard.getAttribute(Core.METACARD_OWNER) != null) {
+            requiredAttributes.put("owner",
+                    (String) metacard.getAttribute(Core.METACARD_OWNER)
+                            .getValue());
+        } else {
+            return null;
+        }
+        return requiredAttributes;
+    }
+
+    /**
+     * Returns all associated metacards given two relatable attributes.
+     *
+     * @param primaryFilterAttribute   The primary attribute to search against
+     * @param secondaryFilterAttribute The secondary attribute to search against
+     * @param primaryAttributeValue    A primary attribute value
+     * @param secondaryAttributeValue  A secondary attribute value
+     * @return A set of metacards, filtered by the given attributes and it's values
+     * @throws UnsupportedQueryException  Error
+     * @throws SourceUnavailableException Error
+     * @throws FederationException        Error
+     */
+    public List<Metacard> getAssociatedMetacardsByTwoAttributes(String primaryFilterAttribute,
+            String secondaryFilterAttribute, String primaryAttributeValue,
+            String secondaryAttributeValue)
+            throws UnsupportedQueryException, SourceUnavailableException, FederationException {
+
+        Filter idFilter = filterBuilder.attribute(primaryFilterAttribute)
+                .is()
+                .equalTo()
+                .text(primaryAttributeValue);
+        Filter tagsFilter = filterBuilder.attribute(secondaryFilterAttribute)
+                .is()
+                .equalTo()
+                .text(secondaryAttributeValue);
+        Filter filter = filterBuilder.allOf(idFilter, tagsFilter);
+
+        return resultIterable(catalogFramework,
+                new QueryRequestImpl(new QueryImpl(filter,
+                        1,
+                        PAGE_SIZE,
+                        SortBy.NATURAL_ORDER,
+                        false,
```

Comment:
❗️ I think we do want to request the total result count.

Reply:
changed to `true`